controllers/imagerepository_controller.go

```diff
@@ -123,8 +124,17 @@ func (r *ImageRepositoryReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 			if err != nil {
 				return ctrl.Result{}, err
 			}
+
+			if imageRepositoryFound {
+				log.Info("Found another image repository for", "RepoURL", imageRepository.Status.Image.URL)
+			}
+
+			_, skipDeletion := imageRepository.Annotations[SkipRepositoryDeletionAnnotationName]
+			if skipDeletion {
```

Comment:
We generally require the boolean-like annotations to be set to `"true"`, not just be present. Let's do the same here for consistency?

Reply:
done